.bit/responses/2.3-Week 2 Step 3.md

```diff
@@ -7,42 +7,97 @@ step: 3
 
-The Face API will accept the image and return information about the face, specifically emotions. Watch this video on Microsoft Cognitive Services for an in-depth explanation: http://www.youtube.com/watch?v=2aA8OEZ1wk8 
-</details>
-<br>
 <details>
-<summary>❓ Where do I begin?</summary>
+<summary>:question: How do I create a Giphy API key?</summary>
 </br>
 
-1. Log into your Azure portal, press **Create a Resource**, the **AI + Machine Learning** tab on the left, and finally select **Face** and fill out the necessary information.Record and save the API endpoint and [subscription key](https://docs.microsoft.com/en-us/azure/api-management/api-management-subscriptions)
-</details>
+To create an **API key**, click [here](https://developers.giphy.com/dashboard/) and click Create an App. <br>
+<img width="352" alt="Screen Shot 2021-04-15 at 5 55 16 PM" src="https://user-images.githubusercontent.com/70852990/114943343-d892d200-9e13-11eb-9365-9f6aec7692c3.png">
+
+Select API, **not** SDK! <br>
+<img width="663" alt="Screen Shot 2021-04-15 at 5 55 32 PM" src="https://user-images.githubusercontent.com/70852990/114943349-daf52c00-9e13-11eb-870e-0be1998c2e04.png">
+
+Then, enter the required information. <br>
+<img width="663" alt="Screen Shot 2021-04-15 at 5 55 41 PM" src="https://user-images.githubusercontent.com/70852990/114943355-dd578600-9e13-11eb-8257-1d9b65f8f5f1.png">
+
 <br>
+Finally, click Create App, and your key should be given!
 
-<details>
-<summary>❓ Where can I find the keys?</summary>
- </br>
+<br><br/>
+</details>
 
-* Navigate to the home page on the Micrsoft Azure portal (https://portal.azure.com/#home)
+## Calling in the Giphy API
+<img src="https://media0.giphy.com/media/833TXYqnMw7Go/200w.webp?cid=ecf05e47z1r1e1ax62zp68eeex57a9bt7vwowretds1ue128&rid=200w.webp&ct=g" height="250">
 
-<img width="1440" alt="Screen Shot 2021-02-04 at 4 00 33 PM" src="https://user-images.githubusercontent.com/28051494/106971033-edac0000-6702-11eb-8243-1b5c2318f76d.png">
+GIPHY Translate converts words and phrases to the perfect GIF or Sticker using GIPHY's special sauce algorithm. For example, the word `beard` gets converted to this masterpiece 
 
-* Click on the resource you need the keys for
+Make sure you edit your `emotional.js` function to call the Giphy API and translate the results from the Face API for a gif that matches the dominant emotion from your emotion data.
 
-<img width="1438" alt="Screen Shot 2021-02-04 at 4 00 49 PM" src="https://user-images.githubusercontent.com/28051494/106971035-ef75c380-6702-11eb-965b-c3ef7b5a7574.png">
+Don't forget to have fun with the API – customize your call using the different parameters. Remember to use the gif and not the sticker endpoint!
 
-* On the left menu bar, locate the Resource Management section and click on "Keys and Endpoint"
+<details>
+<summary>:exclamation: Can I get a more detailed list of steps?</summary>
+  </br>
+  
```

Comment:
There needs to be specific code snippets on how to determine which emotion is most dominant